packages/lumx-icons/override/generate.js

```diff
@@ -0,0 +1,172 @@
+}
+
+/**
+ * Generate TS  aliases
+ */
+async function generateTSAliases() {
+    const importsTS = [];
+    const outTS = [];
+
+    for (const [name, aliases] of Object.entries(aliasesConfig)) {
+        const mainName = camelCase(`mdi-${name}`);
+        importsTS.push(mainName);
+        for (const alias of aliases) {
+            outTS.push(`export const ${camelCase(`mdi-${alias}`)} = ${mainName};`);
+        }
+    }
+
+    importsTS.sort();
+
+    // Output TS
+    outTS.unshift(`export * from './${OUT_OVERRIDE_NAME}';`);
+    outTS.unshift(`import {\n    ${importsTS.join(',\n    ')},\n} from './${OUT_OVERRIDE_NAME}';`);
+    const outTSFile = path.resolve(GENERATED_DIR, `${OUT_ALIASES_NAME}.ts`);
+    await fs.writeFile(outTSFile, `${outTS.join('\n\n')}\n`);
+}
+
+/**
+ * Generate SCSS aliases
+ */
+async function generateSCSSAliases() {
+    const outSCSS = [];
+
+    for (const [name, aliases] of Object.entries(aliasesConfig)) {
+        const aliasesSCSS = [];
+        for (const alias of aliases) {
+            aliasesSCSS.push(`.mdi-${alias}`);
+        }
+        outSCSS.push(`${aliasesSCSS.join(',\n')} {\n    @extend .mdi-${name};\n}`);
+    }
+
+    // Output SCSS
+    const outSCSSFile = path.resolve(GENERATED_DIR, `${OUT_ALIASES_NAME}.scss`);
+    await fs.writeFile(outSCSSFile, `${outSCSS.join('\n\n')}\n`);
+}
+
+/**
+ * Merge MDI font with override font
+ */
+async function generateMergedFont() {
+    const woff2InitPromise = woff2.init();
+    const mdiFontPromise = fs.readFile(MDI_FONT_PATH);
+    const overrideFontPromise = fs.readFile(OVERRIDE_FONT_PATH);
+
+    const overrideCodePoints = Object.values(await configPromise)
+        .map(({ cssCode, replace }) => replace && parseCSSCode(cssCode))
+        .filter(Boolean);
+    const allMDICodePoints = Font.create(await mdiFontPromise, { type: 'ttf' })
+        .find({ filter: () => true })
+        .flatMap(({ unicode }) => unicode);
+    const subset = without(allMDICodePoints, ...overrideCodePoints);
+
+    // Only load MDI icons that are not override
+    const mdiFont = Font.create(await mdiFontPromise, { type: 'ttf', subset });
+    const overrideFont = Font.create(await overrideFontPromise, { type: 'ttf' });
+    mdiFont.merge(overrideFont);
+
+    await woff2InitPromise;
+    const types = ['eot', 'ttf', 'woff', 'woff2'];
+    await Promise.all(
+        types.map((type) => {
+            const filePath = path.resolve(GENERATED_FONT_DIR, `${MDI_FONT_NAME}.${type}`);
+            return fs.writeFile(filePath, mdiFont.write({ type }));
+        }),
+    );
+}
+
+/** Clean up the tmp dir */
+const cleanUp = () => fs.rm(GENERATED_TMP_DIR, { recursive: true, force: true });
+
+generateOverrideFontAndSVG()
+    .then(() =>
+        Promise.all([
+            generateMergedFont(),
+            generateTSOverride(),
+            generateSCSSOverride(),
+            generateTSAliases(),
+            generateSCSSAliases(),
+        ]),
+    )
+    .then(cleanUp);
```

Comment:
Maybe a couple of comments here, explaining what each functions does will make it easier to understand what the script does. Each function is a bit cryptic, they do a lot of file reading, parsing etc, that is not easy to comprehend by just reading them.